public/src/components/channelManagement/campaigns/CampaignsForm.tsx

```diff
@@ -53,6 +54,12 @@ export interface Campaign {
 }
 export type Campaigns = Campaign[];
 
+export const unassignedCampaign = {
+  name: unassignedCampaignLabel,
+  nickname: 'TESTS NOT IN A CAMPAIGN',
```

Comment:
this list of tests might get very large! Something to keep an eye on

Reply:
Agreed. I want this to be a temporary fix that we can rip out once we have added search functionality to RRCP